manage_breast_screening/mammograms/tests/jinja2/test_special_appointment_banner.py

```diff
@@ -38,25 +38,25 @@ def test_special_appointment_banner_with_change_link(jinja_env, presenter):
                 Special appointment
             </span></h3>
 
-            <dl class="nhsuk-summary-list nhsuk-summary-list">
+            <dl class="nhsuk-summary-list special-appointment-banner">
```

Comment:
Would you mind if we followed [this namespace advice](https://github.com/nhsuk/nhsuk-frontend/blob/main/docs/contributing/coding-standards.md#class-naming-convention)?

>If you are building components for your own application or framework you should use a different prefix, for example `.app-` or the initials of your department.

```suggestion
            <dl class="nhsuk-summary-list app-special-appointment-banner">
```